executor/infoschema_reader_test.go

```diff
@@ -869,6 +869,14 @@ func TestNullColumns(t *testing.T) {
 		Check(testkit.Rows("def test v_test type 1 <nil> YES binary 0 0 <nil> <nil> <nil> <nil> <nil> binary(0)   select,insert,update,references  "))
 }
 
+func TestColumnsCopyable(t *testing.T) {
+	store := testkit.CreateMockStore(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+	tk.MustExec("create table cc like information_schema.columns;")
+	tk.MustExec("insert into cc select * from information_schema.columns;")
```

Comment:
This test case is too small. so the startup of this test case will take longer than this test. I suggest it can merge this test case into ```TestCreateTableWithLike```.